specification/guestconfiguration/resource-manager/Microsoft.GuestConfiguration/stable/2018-11-20/guestconfiguration.json

```diff
@@ -0,0 +1,784 @@
+          "default": "False",
+          "description": "Set this to true to automatically reboot the node after a configuration that requires reboot is applied. Otherwise, you will have to manually reboot the node for any configuration that requires it. The default value is false. To use this setting when a reboot condition is enacted by something other than DSC (such as Windows Installer), combine this setting with the xPendingReboot module.",
+          "x-ms-enum": {
+            "name": "rebootIfNeeded",
+            "modelAsString": true
+          },
+          "enum": [
+            "True",
+            "False"
+          ]     
+        },
+        "configurationModeFrequencyMins": {
+          "type": "number",
+          "readOnly": true,
+          "default": 15,
+          "description": "How often, in minutes, the current configuration is checked and applied. This property is ignored if the ConfigurationMode property is set to ApplyOnly. The default value is 15."
+        }
+      },
+      "description": "Configuration setting of LCM (Local Configuration Manager)."
+    },
+    "GuestConfigurationAssignmentProperties": {
+      "properties": {
+        "guestConfiguration": {
+          "$ref": "#/definitions/GuestConfigurationNavigation",
+          "description": "The guest configuration to assign."
+        },
+        "complianceStatus": {
+          "type": "string",
+          "readOnly": true,
+          "description": "A value indicating compliance status of the virtual machine for the assigned guest configuration.",
+          "x-ms-enum": {
+            "name": "complianceStatus",
+            "modelAsString": true
+          },
+          "enum": [
+            "Compliant",
+            "NonCompliant",
+            "Pending"
+          ]
+        },
+        "lastComplianceStatusChecked": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Date and time when last compliance status was checked.",
+          "x-nullable": true
+        },
+        "latestReportId": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Id of the latest report for the guest configuration assignment. ",
+          "x-nullable": true
+        },
+        "context": {
+          "type": "string",
+          "description": "The source which initiated the guest configuration assignment. Ex: Azure Policy"
+        },
+        "assignmentHash": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Combined hash of the configuration package and parameters.",
+          "x-nullable": true
+        },
+        "provisioningState": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The provisioning state, which only appears in the response.",
+          "x-nullable": true,
+          "x-ms-enum": {
+            "name": "provisioningState",
+            "modelAsString": true
+          },
+          "enum": [
```

Comment:
With only "Succeeded" as a value, it looks like provisioningState is spurious.

Reply:
added other enum values